plant-image/plant-image.py

```diff
@@ -1,106 +0,0 @@
-#!/usr/bin/python
-
-import logging
-import os
-import shutil
-import sys
-import tempfile
-import warnings
-from datetime import datetime, timedelta
-
-import numpy as np
-from netCDF4 import Dataset
-from PIL import Image
-from scipy import misc
-
-# wget https://imos-data.s3-ap-southeast-2.amazonaws.com/IMOS/SRS/SST/ghrsst/L3U/n19/2015/20151231145731-ABOM-L3U_GHRSST-SSTskin-AVHRR19_D-Des.nc
-# ./plant-image.py 20151231145731-ABOM-L3U_GHRSST-SSTskin-AVHRR19_D-Des.nc gone-skiing-srs.png 19500101000000-ABOM-L3U_GHRSST-SSTskin-AVHRR19_D-Des.nc
```

Comment:
We did remove this image from the portal, right??

Reply:
yes